### DISCUSSION TITLE
Bootsrap scss not show on ftp

### DISCUSSION THREAD

**Discussion by Jumbo125** [twbs/bootstrap#39321] [Q&A] [answered]
Hello everyone

does bootstrap creates a temporary scss folder???

I was asked to view an existing page. The website works well online. Using FTP I have the data offline on the xampp server and lo and behold, the display is different.
It quickly becomes apparent that some scss files are not loaded offline. Namely the bootstrap scss file. These should be in the directory: scss
However, this folder does not exist on the server.
Therefore I think that bootstrap only creates this temporarily. can this be?
Therefore it could be a problem with the folder rights on the xampp server local.
Thank you!

**Comment by JamesVanWaza** [comment]
@Jumbo125 are you using (https://getbootstrap.com/docs/5.3/getting-started/download/) - Which Package Manager are you using? That will determine if you use SCSS Files.

**Comment by patkon** [comment, marked as answer]
Just for better understanding: The SCSS files are not needed on the live page. With the SCSS files you can generate your own Bootstrap CSS file if that is needed (i.e. if you want to make changes to the default Bootstrap CSS).

If the page uses a custom Bootstrap version, you will of course need these SCSS files to extend or change the visual appearance of the page. So, if the page was generated with modified scss files.

To turn the SCSS files into a working Bootstrap CSS, you need additional software or similar.
I personally know and use only CodeKit - but there are many others.
